tap_gitlab/streams.py

```diff
@@ -54,6 +75,111 @@ def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
             "project_path": context["project_path"],
         }
 
+    def get_url_params(
+        self, context: Optional[dict], next_page_token: Optional[Any]
+    ) -> Dict[str, Any]:
+        """Return a dictionary of values to be used in URL parameterization.
+
+        Projects endpoint can include license info if explicitly requested.
+        """
+        params = super().get_url_params(context, next_page_token)
+        if self.config.get("include_license_info_for_projects", False):
+            params["license"] = True
+        return params
```

Comment:
Can we default this to `true` and remove the config option?

Reply:
Good idea. Done.